tests/lib/exporters/timesketch_tam.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests the Threaded Timesketch exporter."""
+
+import unittest
+
+import mock
+
+from dftimewolf import config
+from dftimewolf.lib import state
+from dftimewolf.lib.containers import containers
+from dftimewolf.lib.exporters import timesketch_tam as timesketch
+from dftimewolf.lib import errors
+
+
+class TimesketchExporterThreadedTest(unittest.TestCase):
+  """Tests for the Threaded Timesketch exporter."""
+
+  def testInitialization(self):
+    """Tests that the processor can be initialized."""
+    test_state = state.DFTimewolfState(config.Config)
+    timesketch_exporter = timesketch.TimesketchExporterThreaded(test_state)
+    self.assertIsNotNone(timesketch_exporter)
+
+  # pylint: disable=invalid-name
+  @mock.patch('dftimewolf.lib.timesketch_utils.GetApiClient')
+  def testSetupWithSketchId(self, mock_GetApiClient):
+    """Tests the SetUp function."""
+    mock_sketch = mock.Mock()
+    mock_sketch.id = 1234
+    mock_sketch.my_acl = ['write']
```

Comment:
Isn't `mock.Mock()` taking arbitrary arguments such that you can inline mocks? E.g. `mock.Mock(id=1234, my_acl=['write'])`. Note that I remember seeing some flaky behaviour when experimenting with those, but perhaps in your case it'd work and reduce a bit the code.